e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -1,16 +1,17 @@
-import _ from "underscore";
 import {
   createAction,
   describeEE,
   describeWithSnowplow,
   enableTracking,
   expectGoodSnowplowEvents,
   expectNoBadSnowplowEvents,
+  modal,
   popover,
   resetSnowplow,
   restore,
   setActionsEnabledForDB,
   setTokenFeatures,
+  summarize,
```

Comment:
The arrow focus state is cut off and hardly visible.

In the video I'm using <kbd>Tab</kbd> and <kbd>Shift</kbd> + <kbd>Tab</kbd>.

https://github.com/metabase/metabase/assets/6830683/dc64eaef-8782-4605-8ae6-adb9ea6f6a29

Reply:
Fixed